crypto/keys/internal/ecdsa/privkey.go

```diff
@@ -0,0 +1,69 @@
+package ecdsa
```

Comment:
I understand we're keeping the cgo version for secp256k1 because it's faster. Can we remove the custom code in the nocgo version? It should just call functions from this package.

can be done in a future PR too.

Reply:
Good idea - let's remove the secp256k1 custom code in other issue. -> https://github.com/cosmos/cosmos-sdk/issues/8766